src/Storages/StorageDistributed.h

```diff
@@ -43,7 +43,7 @@ class StorageDistributed final : public shared_ptr_helper<StorageDistributed>, p
     friend class StorageSystemDistributionQueue;
 
 public:
-    ~StorageDistributed() override;
+    virtual ~StorageDistributed() override;
```

Comment:
According to our style, if you write `override` you don't need to write `virtual`.